master/getting-started/bare-metal/bare-metal.md

```diff
@@ -479,28 +480,80 @@ outside the cluster:
    then be accepted or dropped according to workload policy (if it is going to
    a local workload) or to normal host endpoint policy (if not).
 
+## Policy on forwarded traffic
+If `applyOnForward` is `false`, host endpoint policy applies to traffic to/from
+ local processes only. 
+
+If `applyOnForward` is `true`, host endpoint policy applies to forwarded traffic:
+- Incoming traffic from a host endpoint to a local workload (container/pod/VM).
+- Outgoing traffic from a local workload to a host endpoint.
+- Traffic from one host endpoint that is forwarded to another host endpoint.
+
+By default, `applyOnForward` is `false`. 
+
+Untracked policies and pre-DNAT policies must have `applyOnForward` set to `true`
+because they apply to all forwarded traffic.
+
+Forwarded traffic is allowed by default. In other words, if a host endpoint is
+configured, but there are no rules that explicitly allow or deny a particular
+forwarding flow (in any of policies with applyOnForward true that apply to that
+host endpoint), that flow is still allowed. This is different from how Calico
+treats traffic to or from a local process: traffic to or from a local process is
+denied by default, if a host endpoint is configured but there is no applicable
+policy that explicitly allows that traffic.
+
+> **Note**: Calico's handling of host endpoint policy has changed, since before
+> Calico v2.7.0, in two ways:
+> - It will not apply at all to forwarded traffic, by default. If you have existing
```

Comment:
You should add an "an" in here. So it would be "have existing" --> "have an existing"